data/core/titleview.lua

```diff
@@ -66,7 +66,7 @@ function TitleView:draw_window_title()
   local h = style.font:get_height()
   local ox, oy = self:get_content_offset()
   local color = style.text
-  local x, y = ox + style.padding.x, oy + style.padding.y
+  local x, y = ox +style.padding.x, oy + style.padding.y
```

Comment:
```suggestion
  local x, y = ox + style.padding.x, oy + style.padding.y
```

This is probably a misclick